app/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>WebsiteMy</title>
```

Comment:
Can we please change this to something like `My Site | Real Dev Squad` ?
Suggestions are welcome 😄